RubeusCore/Source/engine.cpp

```diff
@@ -105,20 +106,22 @@ namespace Rubeus
 			item->begin();
 		}
 
-		// TODO: Add Message system and send load level calls asynchronously
+		// Send load level calls asynchronously
+		std::thread th(&RMessageSystem::evaluateMessages, *m_MessageSystem);
+		th.detach();
```

Comment:
It would be better to add some sort of a default message that goes through whenever the engine starts to run. Maybe like a `system_ok` signal to the engine